xml-csharp/XMLDeserializationInCsharp/XMLDeserializationInCsharp/Book.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace XMLDeserializationInCsharp
+{
+    public class Book
+    {
+        [XmlElement("Title")]
+        public string Title { get; set; }
+
+        [XmlElement("Author")]
+        public string Author { get; set; }
```

Comment:
This would also be a good candidate showing how to use a record. Instead of doing the Person class, we could use this one and show how to assign attributes in a positional record statement.
`public record Book([property: XmlElement("Title")] string Title, ...)`

Reply:
Hi @jshergal. Thanks for these suggestions. They are great as always. I will talk to the author to make these changes. Also, I would like to see class deserialization as well, so, we can just add a simple section for record deserialization and apply your changes.